src/view.js

```diff
@@ -0,0 +1,51 @@
+function runView() {
+  const calc = document.querySelector('.calc');
+
+  const calcScreen = document.createElement('div');
+  calcScreen.className = 'calc-screen';
+  calc.prepend(calcScreen);
+
+  const output = document.createElement('p');
+  output.textContent = '0';
+  calcScreen.appendChild(output);
+
+  const buttonsBlock = document.createElement('div');
+  buttonsBlock.className = 'buttons';
+  calc.append(buttonsBlock);
+
+  const buttons = [
+    { name: 'ac', class: 'ac bg-grey' },
+    { name: '+/-', class: 'plus-minus bg-grey' },
+    { name: '%', class: 'percent bg-grey' },
+    { name: '/', class: 'percent bg-grey' },
+    { name: '7', class: 'seven' },
+    { name: '8', class: 'eigth' },
+    { name: '9', class: 'nine' },
+    { name: '*', class: 'myltiply bg-orange' },
+    { name: '4', class: 'four' },
+    { name: '5', class: 'five' },
+    { name: '6', class: 'six' },
+    { name: '-', class: 'minus bg-orange' },
+    { name: '1', class: 'one' },
+    { name: '2', class: 'two' },
+    { name: '3', class: 'three' },
+    { name: '+', class: 'plus bg-orange' },
+    { name: '0', class: 'zero' },
+    { name: '.', class: 'dot' },
+    { name: '=', class: 'equal bg-orange' },
+  ];
+
+  const numericBtn = document.createDocumentFragment();
+
+  // eslint-disable-next-line no-restricted-syntax
```

Comment:
read more about rule https://stackoverflow.com/questions/73127625/alternatives-for-using-forinstatement-is-not-allowed

better use destructing